app/services/chatbot_service.rb

```diff
@@ -0,0 +1,22 @@
+require 'httparty'
+
+class ChatbotService
+  attr_reader :url
+
+  def initialize(url)
+    @url = url
+  end
+
+  def post_message(options)
+    message = "Subject: #{options[:subject]}<br/> #{options[:message]}"
```

Comment:
Let's keep message formatting out of this class. Perhaps makes sense to add something like "MessageTempate" class